### PR TITLE
[WIP] Initial PR for helpdesk module

### PR DESCRIPTION
**Pull request by aitorbouzas**
This is an initial PR of the helpdesk base module. There are yet several things to implement and to test (that's why it's a WIP).

It will help us to have some reviewers too.

Checklist:

- [x] Mail integration - https://github.com/aitorbouzas/OCA-helpdesk/pull/5
- [x] Readme usage section
- [x] Dashboard menu
- [x] Partners smart button for tickets
- [ ] Mailing templates for closed ticket, and stage changed
- [ ] Rename module - https://github.com/aitorbouzas/OCA-helpdesk/pull/15
- [ ] Functional testing
- [ ] Tests (codecov)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Is this going to be continued? You should do peer review here between the members of the table.

**Comment by aitorbouzas** [comment]
Of course this will be continued. We stated in the table the minimum tasks to be done until this base module is considered usable, and those are the ones specified in the PR comment up here. This tasks are not that hard, and I hope I can work on them soon enough.

**Comment by pedrobaeza** [comment]
Great. Please organize for finishing them and see the first release of this soon.

**Comment by aitorbouzas** [comment]
Adding mail integration.
Help needed for testing every possible line of code and for mailing templates.

**Comment by pedrobaeza** [comment]
@agenterp can you review this?

**Comment by aitorbouzas** [comment]
This -> https://github.com/aitorbouzas/OCA-helpdesk/pull/5

**Comment by agent-z28** [comment]
I'll review it soon, one idea: 
What do you think of integrating with this?
https://github.com/OCA/contract/blob/11.0/agreement/models/res_config_settings.py#L29

**Comment by aitorbouzas** [comment]
@agenterp It would be nice, there are plenty of integrations to be done. Let's unite efforts in the development of the base module first.

**Comment by agent-z28** [comment]
@aitorbouzas of course, i just wanted to add it to the todolist

**Review by agent-z28 — APPROVED** [review]
Hi, i checked the code and I checked the functionality of the mdules from the build.
For me it looks propper. 
I didnt check the security rights.
Still we need the mail templates

**Comment by aitorbouzas** [comment]
Thank you! @agenterp

**Review by carlos-domatix — APPROVED** [review]

**Review by miguel-domatix — APPROVED** [review]

**Review by samuel-domatix — APPROVED** [review]

**Review by alvaro-domatix — APPROVED** [review]

**Comment by pedrobaeza** [comment]
This merge operation has ruined all the commit history and now there are conflicts plaining it:

Selección_011

I'm dealing with the consequences solving conflicts, but please NEVER do this kind of conflict resolution.

**Comment by aitorbouzas** [comment]
I'm sorry Pedro, you now have to deal with my inexperience...
I think this branch shouldn't be merged yet, name's not changed and there are some things pending to do...

**Comment by pedrobaeza** [comment]
I'm doing the renaming, and that's why I found the problem. We are going to merge it for allowing other PRs improving features. I'm explicitly tagging the module as Alpha.

I also miss README content explaining usage, configuration, and so.

File: helpdesk/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Helpdesk',
+    'summary': """
+        Helpdesk""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'After-Sales',
+    'author':   'AdaptiveCity, '
```
Comment (pedrobaeza) [inline helpdesk/__manifest__.py:10]: @emagdalenaC2i this alignment is not suitable in Python. Take it into account for next time. I'm correcting it on my rebase.

**Comment by pedrobaeza** [comment]
Now I'm seeing the README, but the commit is not accessible due to merge operation. I continue digging...

**Comment by pedrobaeza** [comment]
@aitorbouzas your repository is in read-only mode. Can you switch it for pushing the version with the rename and latest changes before merging initial version?

File: helpdesk/__manifest__.py (hunk 1 above)
Comment (emagdalenaC2i) [reply helpdesk/__manifest__.py:10]: Ok, noted

**Comment by aitorbouzas** [comment]
@pedrobaeza Do you mean the option "Restrict editing to collaborators only"?
I just deactivated it

**Comment by pedrobaeza** [comment]
No, it's your whole fork that is readonly. I'm opening a new PR any way for continuing the work: #24 

Can you all confirm that everything is correct in the other PR for proceeding to merge?

**Comment by aitorbouzas** [comment]
@pedrobaeza Thanks for the effort!

**Pull request closed without merging**